doc/onboarding-extras.md

```diff
@@ -25,6 +25,7 @@
 | `src/async-wrap.*` | @nodejs/async_hooks |
 | `src/node_api.*` | @nodejs/n-api |
 | `src/node_crypto.*` | @nodejs/crypto |
+| `src/inspector_*`, `lib/inspector.js` | @nodejs/v8-inspector |
```

Comment:
@Trott Ah totally makes sense!! Done. Added a second commit to. Is that alright ?